app/gzac/src/main/java/com/ritense/gzac/listener/ApplicationReadyEventListener.kt

```diff
@@ -107,23 +107,11 @@ class ApplicationReadyEventListener(
 
     fun createPlugins() {
         try {
-            val zakenApiAuthenticationPluginId = createZakenApiAuthenticationPlugin()
-            val zakenApiPluginId = createZakenApiPlugin(zakenApiAuthenticationPluginId)
-            createBesluitenApiPlugin(zakenApiAuthenticationPluginId)
-            createCatalogiApiPlugin(zakenApiAuthenticationPluginId)
-            val documentenApiPluginId = createDocumentenApiPlugin(zakenApiAuthenticationPluginId)
-            val notificatiesApiAuthenticationPluginId = createNotificatiesApiAuthenticationPlugin()
-            val notificatiesApiPluginId = createNotificatiesApiPlugin(notificatiesApiAuthenticationPluginId)
-            val objectenApiAuthenticationPluginId = createObjectenApiAuthenticationPlugin()
-            val objectenApiPluginId = createObjectenApiPlugin(objectenApiAuthenticationPluginId)
-            val objecttypenApiAuthenticationPluginId = createObjecttypenApiAuthenticationPlugin()
-            val objecttypenApiPluginId = createObjecttypenApiPlugin(objecttypenApiAuthenticationPluginId)
-            val bezwaarConfigurationId = createBezwaarObjectManagement(objecttypenApiPluginId, objectenApiPluginId)
-            val taakConfigurationId = createTaakObjectManagement(objecttypenApiPluginId, objectenApiPluginId)
-            createBomenObjectManagement(objecttypenApiPluginId, objectenApiPluginId)
```

Comment:
Can you also make an object-management auto deployment for `bomen.json`?

Also, there should now be a lot of unused functions in this ApplicationReadyEventListener. They can all be removed

Reply:
what do you mean? i already did for all of that were in that class